### DISCUSSION TITLE
export the individual f3z files on thingiverse

### DISCUSSION THREAD

**Discussion by ogg1e** [mebs/CM4-NAS#33] [General]
I see that you exported the design to a sketchup (skp) file, but that contains the entire case and not the 3 individual pieces (stl files) you uploaded. 

I wanted to make some changes to the main body, but I cannot since I don't know where you split the case into 3 pieces. 

If you were to create 3 skp files for the 3 different components you uploaded, that would be great.

Thanks @mebs

**Comment by mebs** [comment]
Have you tried opening the Fusion 360 project? You should be able to do whatever you want from there.

**Reply by ogg1e** [reply]
I was able to get a demo version of f360 and export the individual components from there.

**Reply by mebs** [reply]
You could get a Fusion 360 copy for hobbyists/personal use, it's free and you have everything you need in there for 3D printing.
They intentionally make it a bit hard to find, but if you look specifically for it you'll find it.
https://www.autodesk.ca/en/products/fusion-360/personal
![image](https://user-images.githubusercontent.com/2614134/158840531-e6aad943-2de1-45c7-9a97-dc38605ba650.png)
